ui/components/app/confirm-page-container/confirm-page-container-content/confirm-page-container-summary/confirm-page-container-summary.component.js

```diff
@@ -116,7 +116,14 @@ const ConfirmPageContainerSummary = (props) => {
         <div className="confirm-page-container-summary__title">
           {renderImage()}
           {!hideTitle ? (
-            <div className="confirm-page-container-summary__title-text">
+            <div
+              className={
+                title && title.length < 10
+                  ? 'confirm-page-container-summary__title-text'
+                  : 'confirm-page-container-summary__title-text-long'
+              }
+              title={title}
+            >
```

Comment:
An option here is to use `Typography` component.

Reply:
done here: https://github.com/MetaMask/metamask-extension/pull/13788/commits/86826e7812064f1be75d04d852659b13cd212106